src/io/opendss.jl

```diff
@@ -194,20 +194,132 @@ function dss2tppm_load!(tppm_data::Dict, dss_data::Dict, import_all::Bool)
             name, nodes = parse_busname(defaults["bus1"])
 
             kv = defaults["kv"]
+
             expected_kv = tppm_data["basekv"] / sqrt(tppm_data["conductors"])
             if !isapprox(kv, expected_kv; atol=expected_kv * 0.01)
                 Memento.warn(LOGGER, "Load has kv=$kv, not the expected kv=$(expected_kv). Results may not match OpenDSS")
             end
 
             loadDict["name"] = defaults["name"]
             loadDict["load_bus"] = find_bus(name, tppm_data)
-            loadDict["pd"] = PMs.MultiConductorVector(parse_array(defaults["kw"] / 1e3, nodes, nconductors))
-            loadDict["qd"] = PMs.MultiConductorVector(parse_array(defaults["kvar"] / 1e3, nodes, nconductors))
+
```

Comment:
I believe this now adds the follow fields to the load dict, correct me if I'm mistaken: `conn`, `model`, `vnom_kv`

Reply:
Correct. I think @ccoffrin will prefer renaming `vnom_kv` to `vm_nom`, similar to the transformer data model.